codecov_cli/helpers/git.py

```diff
@@ -92,14 +92,24 @@ def parse_git_service(remote_repo_url: str):
         return None
 
 
-def is_fork_pr(pr_num, slug, service):
+def is_fork_pr(pull_dict):
     """
-    takes in pull request number, slug in the owner/repo format, and the git service e.g. github, gitlab etc.
+    takes in dict: pull_dict
     returns true if PR is made in a fork context, false if not.
     """
+    return (
+        True
+        if pull_dict and pull_dict["head"]["slug"] != pull_dict["base"]["slug"]
+        else False
+    )
```

Comment:
rather than use a ternary here you could do this in one line like:
```
return pull_dict and pull_dict["head"]["slug"] != pull_dict["base"]["slug"]
```

Reply:
yup missed that. Thank you I'll update it